README.md

```diff
@@ -13,5 +13,11 @@ The main focus of this project will encompass the details page which will be com
 First run:
 `Run npm install`
 
+Second rename:
+`src/config.ex.js`
+to
+`src/config.js`
+and change 'ur_key' to your auth token
```

Comment:
I agree with Andrew, we should probably change 'ur_key' to something more professional, I guess.